content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
```

Comment:
I'd make GatewayClass, Gateway and HTTPRoute be hyperlinks to https://gateway-api.sigs.k8s.io/beta1/api-types/gatewayclass/ and so on (remember to produce those docs for the release, too).

Reply:
```suggestion
beta with the `v0.5.0` release of the API. In addition to [GatewayClass](https://gateway-api.sigs.k8s.io/api-types/gatewayclass/),
[Gateway](https://gateway-api.sigs.k8s.io/api-types/gateway/) and [HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/) graduating to beta, this release introduces several
```